docs/en-US/component/mention.md

```diff
@@ -0,0 +1,143 @@
+| ------------------------------------ | -------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------- | ---------- |
+| options                              | mention options list                                                                   | ^[array]`MentionOption[]`                                                    | []         |
+| prefix                               | prefix character to trigger mentions. The string length must be exactly 1              | ^[string]                                                                    | `'@'`      |
+| split                                | character to split mentions. The string length must be exactly 1                       | ^[string]                                                                    | `' '`      |
+| filter-option                        | customize filter option logic                                                          | ^[false] \| ^[Function]`(pattern: string, option: MentionOption) => boolean` | -          |
+| placement                            | set popup placement                                                                    | ^[string]`'bottom' \| 'top'`                                                 | `'bottom'` |
+| whole                                | when backspace is pressed to delete, whether the mention content is deleted as a whole | ^[boolean]                                                                   | `false`    |
+| check-is-whole                       | when backspace is pressed to delete, check if the mention is a whole                   | ^[Function]`(pattern: string, prefix: string) => boolean`                    | -          |
+| loading                              | whether the dropdown panel of mentions is in a loading state                           | ^[boolean]                                                                   | `false`    |
+| model-value / v-model                | input value                                                                            | ^[string]                                                                    | -          |
+| popper-class                         | custom class name for dropdown panel                                                   | ^[string]                                                                    | -          |
+| popper-options                       | [popper.js](https://popper.js.org/docs/v2/) parameters                                 | ^[object] refer to [popper.js](https://popper.js.org/docs/v2/) doc           | -          |
+| [input props](./input.md#attributes) | -                                                                                      | -                                                                            | -          |
```

Comment:
@tolking Do we need to rely on the types in the document when generating type files? If so, is there something wrong with this way of writing?

Reply:
The Mention component in the web-types.json file generated in this way will lack Input related type prompts.

Since this type of conversion will be removed in the future, I think writing it this way is also acceptable now.

If a user of WebStorm mentions this issue, we can recommend that they enable Volar.

WDYT

Reply:
I think it's OK. But is it OK to generate a JSON file with type `-`?

Reply:
`-` will be converted to undefined